multiversx_sdk_core/transaction_factories/transaction_factory_config.py

```diff
@@ -25,3 +27,12 @@ def __init__(self, chain_id: IChainID) -> None:
         self.gas_limit_store_per_byte: IGasLimit = 50_000
         self.issue_cost: ITransactionValue = 50_000_000_000_000_000
         self.esdt_contract_address: IAddress = Address.from_bech32("erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u")
+
+        # Configuration for delegation operations
+        self.gas_limit_stake = 5000000
+        self.gas_limit_unstake = 5000000
+        self.gas_limit_unbond = 5000000
+        self.gas_limit_create_delegation_contract = 50000000
+        self.gas_limit_delegation_operations = 1000000
+        self.additional_gas_limit_per_validator_node = 6000000
+        self.additional_gas_for_delegation_operations = 10000000
```

Comment:
Perhaps also add underscores to these values as well. Helps with consistency and makes the values more readable.

Reply:
Indeed, fixed.